src/plugins/intel_cpu/src/transformations/snippets/x64/op/brgemm_cpu.cpp

```diff
@@ -22,11 +23,12 @@ BrgemmCPU::BrgemmCPU(const Output<Node>& A, const Output<Node>& B, const Type ty
     set_input_port_descriptor({0, offset_a}, 0);
     set_input_port_descriptor({0, offset_b}, 1);
     set_output_port_descriptor({0, offset_c}, 0);
-    constructor_validate_and_infer_types();
+    custom_constructor_validate_and_infer_types(layout_a, layout_b, layout_c);
```

Comment:
Minor: since the layouts are passed by value, it makes sense to std::move them:
```suggestion
    custom_constructor_validate_and_infer_types(std::move(layout_a), std::move(layout_b), std::move(layout_c));
```

Reply:
Exactly, I missed it in `BrgemmCPU`. Thanks!